js/ui/drop_down_box.js

```diff
@@ -137,6 +137,12 @@ var DropDownBox = DropDownEditor.inherit({
              * @default "change"
              */
 
+            /**
+             * @name dxDropDownBoxOptions.valueFormat
+             * @type function(value)
+             * @type_function_param1 value:string|Array<any>
+             * @type_function_return string
+             */
             valueFormat: function(value) {
```

Comment:
@AlexJameson 
Should we rename this option before publish?
I think the current name can confuse other people. They may decide that this option accept value with the [Format](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/) type.
Another point - description will conflict with other published `valueFormat` option which accept  a value with the `Format`type

Reply:
I suggest changing the option name to valueFormatter, since we already have formatter function and this name describes the actions quite precisely.